cmd/api/src/analysis/azure/queries_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2023 Specter Ops, Inc.
+//
+// Licensed under the Apache License, Version 2.0
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build integration
+// +build integration
+
+package azure_test
+
+import (
+	"context"
+	"github.com/specterops/bloodhound/dawgs/graph"
+	azure2 "github.com/specterops/bloodhound/src/analysis/azure"
+	"github.com/specterops/bloodhound/src/model"
+	"github.com/specterops/bloodhound/src/test/integration"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestAnalysisAzure_GraphStats(t *testing.T) {
+	testCtx := integration.NewGraphTestContext(t)
+	testCtx.DatabaseTest(func(harness integration.HarnessDetails, db graph.Database) error {
+		expectedAgg := model.AzureDataQualityAggregation{
```

Comment:
If there's a more programmatic way to get these counts from the harnesses to make this test less brittle, please lmk. Not a fan hardcode.